MobileWalletTests/MobileWalletTests.swift

```diff
@@ -91,7 +91,31 @@ class MobileWalletTests: XCTestCase {
             XCTFail("Failed to iterate through theme assets")
         }
     }
+    
+    func testDateFallsInWeek() {
+        let testDateString = "2019-10-01"
+        let dateFormatter = DateFormatter()
+        dateFormatter.dateFormat = "yyyy-mm-dd"
+        guard let testDate = dateFormatter.date(from: testDateString) else {
+            XCTFail("Failed to get the date from the string value")
+            return
+        }
+        
+        if DateConfig.dateFallsInCurrentWeek(date: testDate) == true {
+            XCTFail("Date does not fall in current week. Test Date: \(testDate). Today's Date: \(Date())")
+        }
+    }
+    
+    
+    func testRelativeDayValue() {
+        let testDateString = Date()
+        let dateValue =  DateConfig.getRelativeDayValue(fromDate: testDateString)
+        if dateValue != NSLocalizedString("Today", comment: "") {
```

Comment:
Need to test all possible paths a function could execute. `Today`, `Tomorrow`, and full date